main.js

```diff
@@ -0,0 +1,55 @@
+ var article2="So glad for the prework right now.  Our first group project was nerve-wracking, but so heady.  Our website was a sales site for Temporal Tours, a time-travel tour company concerned less with client safety than with financial gain.  We reached for a highly ridiculous version of our real-life experiences.  One of my partners is a former Latin teacher, too! We parlayed our travel disasters into website gold. One of the main challenges in this project is that we're a person down, since one of our team members dropped from the program, but we've stepped up to create a great site, and I've created my first JavaScript magic!  Seeing those cards appear on the page literally made me laugh out loud with joy!  Hard work pays off....I'm also getting a great teaching workout, since we're all in different places with our understanding.  If I can help out my teammates, I'm there with bells on."
+ var article3="We presented our second group project today, and the experience crystallized exactly what I need to work on and what I do well.  I get along with everyone, communicate clearly, work my posterior off, but I don't advocate for myself or my ideas well enough yet.  I think we came to a good solution eventually, but it felt like talking to a brick wall at times, and it cost us time we didn't have to waste.  I know my partner probably felt exactly the same, so it'll be something to work on in group projects for me. Our second attempt at a similar format of project also awakened the realization that I was too focused on just getting done and didn't spend enough time being creative with code. The day and a half we had to come up with our idea and execute a 5 or 6 page site complete with js, css, and content just wasn't enough for me to produce the quality of work I'm capable of.  Time crunch and creativity together are another point to work on.  I have to remind myself, 'Don't just do what works, take some chances!' Different work ethics also were a point of difficulty this time around.  Not everyone is willing or able to put in the time necessary, and it almost meant presenting an unfinished product version.  I know I have to focus on gentle prodding, encouragement, and otherwise to help light fires where necessary.  Simmering discontent due to different work styles does not make for the most productive work groups, so I will work on that aspect of my communication in groups."
+
+  let blogPosts = [
+
+  {title:"Terrifying but Exhilarating", copy: article1, date:"5/25/17"},
+  
+  {title:"First Group Project", copy: article2, date:"5/30/2017"},
+
+  {title:"Project Redux at Breakneck Speed", copy:article3, date:"6/2/2017"}
+  
+  ];
+
+ var html = "";
+  function printToDiv(message){
+
+  	let blogDiv = document.getElementById("blog");
+  	blogDiv.innerHTML = message;
+  }
+
+function iterateThroughArray(arr){
+for (let i= 0; i<blogPosts.length; i++){
```

Comment:
Watch your indentation. Always be sure to be consistent with nesting within functions.